core/modules/exact_match.cc

```diff
@@ -43,6 +43,12 @@ static inline int is_valid_gate(gate_idx_t gate) {
 }
 
 const Commands ExactMatch::cmds = {
+    {"GetArg", "EmptyArg", MODULE_CMD_FUNC(&ExactMatch::GetArg),
```

Comment:
Could you change the names of commands to `lower_case`, just to be consistent with other commands? Since the command names are used as python method names, the existing code avoids `CamelCase` because of pep8...

Reply:
Sure ... I guess the protobuf names stay CamelCased but should they be, e.g., `ExactMatchConf`?  `ExactMatchConfiguration`?  (Whatever their pattern is, it affects the names retained in the protobuf loader python module.  If that gets complicated maybe I'll use a regexp instead of `.endswith` and literals...)

Reply:
Ah, forgot about the `*Arg` constraint. I think `get_initial_arg` also sounds good.